app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2 class="text-center">Public Recipes</h2>
+<div class="container-fluid recipe-foods">
+<% @public_recipes.each do |recipe| %>
+<div class="card">
+    <div class="d-flex justify-content-around">
+        <div class="p-2">
+             <h3><%= recipe.name %></h3>
+            <h5>By: <%= recipe.user.name%></h5>
+        </div>
+        <div class="d-flex flex-column align-self-center p-2">
+            Total Food Items: <%= recipe.foods.count %>
+            <br>
+            Total Price: <%= recipe.total_price %>
```

Comment:
Well done here as well public recipes are retrieved and listed, but please make sure to implement all requirements in terms of functionalities and stylings as the view needs to follow the wireframe layout and positioning of elements you have the freedom to choose colors and fonts but we need to respect the wireframe, for the functionalities the user should be able to delete their recipes from the public recipes list and all users needs to see recipe details when they click in a public recipe 

![Screenshot from 2024-02-11 14-12-13](https://github.com/AlierPM/Recipedaily/assets/59707859/cf9b8cf2-f884-40a2-981e-b0f3d9490965)

![Screenshot from 2024-02-11 14-13-07](https://github.com/AlierPM/Recipedaily/assets/59707859/113a1656-8301-4597-993b-767c25481c0d)


**Public recipe list:**

Should display a list of all public recipes ordered by newest as in the wireframe.
Should lead to recipe details.
If the user is the owner of the recipe, should allow the user to delete it.